src/main/java/com/example/user_management/service/user/impl/UserServiceImpl.java

```diff
@@ -39,7 +40,8 @@ public User findByUsername(final String username) {
     }
 
     @Override
-    public List<User> getAllUsers() {
-        return userRepository.findAll();
+    public Page<User> getAllUsers(final Integer page, final Integer size) {
+        Pageable pageable = PageRequest.of(page, size);
+        return userRepository.findAll(pageable);
```

Comment:
What is the default sorting option? Will it be a random group of results per page request ?